VogtPayroll2/Employee1.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    abstract class Employee
+    {
+        //For simplicity, use a constant tax rate of 30% to compute the tax amount
+        protected string name;
+        protected int hoursWorked;
+        protected decimal payRate;
+        protected const decimal taxAmount = .30m;
+
+        public Employee(string aName, int aHoursWorked, decimal aPayRate)
+        {
+            name = aName;
+            hoursWorked = aHoursWorked;
+            payRate = aPayRate;
+        }
+        public void DisplayEmployeeInfo(Employee employee)
+        {
+            // The output should display the aName of each Employee, hours worked, hourly rate, overtime pay, regular (gross) pay, tax amount, and net pay.
+            Console.WriteLine($"Employee Name: {employee.name}");
+            Console.WriteLine($"Hours worked: {employee.hoursWorked}");
+            Console.WriteLine($"Hourly rate: {GetPayRate()}");
+            Console.WriteLine($"Overtime pay: {OverTime()}");
+            Console.WriteLine($"Regular (gross) pay: {GetGrossPay()}");
+            Console.WriteLine($"Tax amount: {GetTaxAmount()}");
+            Console.WriteLine($"Net Pay: {GetNetPay()}");
+        }
+
+        public void DisplayNetPayOfAllEmployees(Employee[] employee)
+        {
+            // The average net pay of all employees should also be displayed
+        }
+
+        public decimal FindMinimumAndMaximumNetPay()
+        {
+            // In addition, the program should find the minimum and maximum net pay of all employees as well as sort the employees based on their net pay (ascending order)
+            // The program must combine two sorting techniques; the Selection and Exchange sorts. It will efficiently sort the employees' net pay.
+            // Preferrably without using a data file.
+            decimal netPay = default;
+
+            return netPay;
+        }
```

Comment:
`DisplayNetPayOfAllEmployee` and `FindMinimumAndMaximumNetPay` should be in at the very least 1 new class. For the size of this problem, I'd suggest creating something like `PayrollManager`.

Reply:
How about make all of the other changes to these employee classes then we can talk about how to separate these out in a way so that you can finish the problem.